components/Camp.tsx

```
@@ -1,10 +1,99 @@
-'use client';
+import { PEOPLE_URL } from "@/constants";
+import Image from "next/image";
 
-import React from 'react'
+interface CampProps {
+  backgroundImage: String;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)